internal/install/glob_file_filterer_test.go

```diff
@@ -0,0 +1,96 @@
+// +build unit
```

Comment:
I would call these integration tests since we are interacting with the filesystem.